src/test/resources/missedToken.json

```
@@ -0,0 +1,22 @@
+{
+  "creationDate": "@@TODAY@@",
+  "complaint": {
+    "complaintType": "BIOMETRIC_RESIDENCE_PERMIT",
+    "reference": {
+      "referenceType": "GWF_REF",
+      "reference": "@@MISSED_TOKEN@@"
+    },
+    "reporterDetails": {
+      "applicantType": "APPLICANT",
+      "applicantName": "@@APPLICANT_NAME@@",
+      "applicantNationality": "Denmark",
+      "applicantDob": "2020-10-03",
+      "applicantEmail": "sss@uevptde.com",
+      "applicantPhone": "01772 700806"
```

```suggestion
      "applicantPhone": "0117 4960806"
```
Use [fictional numbers](https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama) instead of real ones.

(Incidentally, `0117` is the area code, `01772` puts the space in the wrong place)

---

Using static safe phone number